lib/main_containers.pm

```diff
@@ -155,6 +155,24 @@ sub load_host_tests_helm {
     }
 }
 
+sub load_host_tests_in_k8s {
```

Comment:
shouldn't this be named `load_image_tests_on_k8s` ? As we are testing the container image, not the host..

Reply:
done